### PR TITLE
Improve and correct ABI names

### LINKED ISSUE #3597: Smart contract Allowance Solidity Interfaces Incorrectly Implemented
The interfaces as described in [HIP-218](https://hips.hedera.com/hip/hip-218) have only been implemented for HTS redirect, not for direct HTS calls.

We should implement a set of methods against the HTS Precompile address (0x167) that includes the token ID as an argument, 
* allowance(address token, address owner, address spender)
* approve(address token, address spender, uint256 amount)
* approveNFT(address token, address to, uint256 tokenId)
* setApprovalForAll(address token, address operator, bool approved)
* isApprovedForAll(address token, address owner, address operator)
*  isApproved(address token, uint256 tokenId)

### PR DESCRIPTION
**Pull request by shemnon**
**Description**:

* Correct the comment names associated with the hashes
* All ABI calls targeting redirect are now named ABI_ID_ERC_<foo>

**Related issue(s)**:

Fixes #3597

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by tdermendzhievv — APPROVED** [review]
LGTM

File: hedera-node/src/main/java/com/hedera/services/exceptions/InvalidTransactionException.java [hunk 1]
```diff
@@ -34,47 +34,42 @@
...
-		return Bytes.of(
-				detail.substring(REVERT_REASON_TAG_START.length(), detail.indexOf(REVERT_REASON_TAG_END)).getBytes());
-	}
-
-	private static String revertingDetail(final String revertReason) {
-		return REVERT_REASON_TAG_START + revertReason + REVERT_REASON_TAG_END;
+		return Bytes.of(detail.getBytes());
 	}
-}
+}
```
Comment (Nana-EC) [inline hedera-node/src/main/java/com/hedera/services/exceptions/InvalidTransactionException.java:75]: nit: new line

**Review by Nana-EC — APPROVED** [review]
LG

**Review by stoqnkpL — APPROVED** [review]
LGTM

**Review by tinker-michaelj — APPROVED** [review]
LGTM!

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**